src/test/java/com/meilisearch/integration/ClientTest.java

```diff
@@ -1,6 +1,13 @@
 package com.meilisearch.integration;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.arrayWithSize;
```

Comment:
Hey @brunoocasali - it's not really about performance or compilation time, in my opinion. I find that it makes it clearer to see which class/method you're calling when they are all imported one by one.

For example:

```java
import static org.some.package.SomeClass.*;
import static org.some.package.AnotherClass.*;

// ...

String a = foo();
```

In the above, it's not clear whether `foo` is imported from `SomeClass` or `AnotherClass`. You could find out quite easily in an IDE, but it's not so clear when reviewing code.

```java
import static org.some.package.SomeClass.foo;

// ...

String a = foo();
```

This one is more explicit and it's clear that `foo` comes from `SomeClass`.

Another thing worth mentioning is that more than one class/package can export methods/classes/types with the same name. So in the first example above, `SomeClass.foo` may be the one you intended, but if `AnotherClass` also had a `public static` method named `foo`, then that one could be the one you ended up calling, without realising it. 

Another, more insidious case, is that it is possible that `AnotherClass` later adds a `foo` method with the same signature as the one in `SomeClass` and suddenly, the code will stop compiling because it is not clear which `foo` was intended by just looking at your signature. This can happen without any code changes in your project too, if `AnotherClass` is in a dependency.

Regardless, this primarily boils down to personal preference. I prefer to be more explicit with imports. My IDE is set up to always bring in explicit imports instead of wildcards, which is why this PR has them. But ultimately, if you prefer wildcards in this project, then that is fine too. I don't mind making that change to align with that.

Reply:
Wow, that's one of the things I love about my job, I can learn a lot with experts from different languages 🤘 

I came from a ruby background and as you guess we don't have that there 😅 

Thanks a lot for your time and patience in explaining that to me!

Reply:
No worries at all. I'm glad it helped 🙂 

Like I said though, this is just my preference and the above is my reasoning and it happens to align with many others who prefer explicit imports over wildcards. On the other hand, there are others which prefer to use wildcard imports since they are much more concise.

If you'd like me to use wildcard imports, I have no objection to making that change. Just let me know 🙂